utility/utils.py

```diff
@@ -0,0 +1,208 @@
+"""
+General utility functions 
+
+"""
+
+import logging
+import os
+import random
+import uuid
+
+from framework import config
+from common_ci_utils.command_runner import exec_cmd
+from framework.ssh_connection_manager import SSHConnectionManager
```

Comment:
order the imports

Reply:
done in https://github.com/red-hat-storage/noobaa-sa-ci/pull/11/commits/bf174e4cf192f95457c6e85267009f7cacb5063a